.NET/Microsoft.Recognizers.Text.Sequence/SequenceRecognizer.cs

```diff
@@ -72,7 +73,7 @@ public IModel GetPhoneNumberModel(string culture = null, bool fallbackToDefaultC
                 return GetModel<PhoneNumberModel>(Culture.Chinese, fallbackToDefaultCulture);
             }
 
-            return GetModel<PhoneNumberModel>(Culture.English, fallbackToDefaultCulture);
+            return GetModel<PhoneNumberModel>(culture, fallbackToDefaultCulture);

```

Comment:
This change effectively breaks every other language. We should implement all cultures, or make any that's not EN and PT still fallback to English.